### PR TITLE
Practical support improvements: Add attachment_url field + fulfilled checkbox, config to display attachment_url

### PR DESCRIPTION
**Pull request by colinxfleming**
I rule and have completed some work on Case Manager that's ready for review!

This pull request makes the following changes:
* adds practical support fields `attachment_url` and `fulfillment`
* adds a config to display `attachment_url` (defaulting to not showing it)

A little cramped, but looks chill:
image
image

It relates to the following issue #s: 
* Bumps #3171 

For reviewer:
* Adjust the title to explain what it does for the notification email to the listserv.
* Tag this PR:
  * `feature` if it contains a feature, fix, or similar. This is anything that contains a user-facing fix in some way, such as frontend changes, alterations to backend behavior, or bug fixes.
  * `dependencies` if it contains library upgrades or similar. This is anything that upgrades any dependency, such as a Gemfile update or npm package upgrade.
* If it contains neither, no need to tag this PR.

### CODE REVIEW

File: app/models/config.rb [hunk 1]
```diff
@@ -24,6 +24,7 @@ class Config < ApplicationRecord
     time_zone: "Time zone to use for displaying dates. Default is Eastern. Valid options are Eastern, Central, Mountain, Pacific, Alaska, Hawaii, Arizona, Indiana (East), or Puerto Rico.",
     procedure_type: "Any kind of distinction in procedure your fund would like to track. Field hides if no options " \
                     "are added here. Please separate with commas.",
+    display_practical_support_attachment_url: 'CAUTION: Whether or not to allow people to enter attachment URLs for practical support entries; for example, a link to a file in Google Drive. Please ensure that any system storing these is properly secured by your fund!',
```
Comment (lomky) [inline app/models/config.rb:27]: i def still feel wary about this, but I think this is the right balance of empowering vs security

**Review by lomky — APPROVED** [review]
Can we update teh [data dictionary](https://github.com/DARIAEngineering/dcaf_case_management/blob/main/docs/DATA_DICTIONARY.md) doc too? I think we've drifted a bit

File: app/models/config.rb (hunk 1 above)
Comment (colinxfleming) [reply app/models/config.rb:27]: I also feel ways about this but yeah I think this is an okay balance

**Comment by colinxfleming** [comment]
bumped the data dict -- merging

**Pull request merged**